interfaces/Portal/src/app/components/update-property-item/update-property-item.component.html

```diff
@@ -25,14 +25,17 @@
   method="POST"
   #updatePropertyForm="ngForm"
 >
-  <div class="input-item update-property-item">
-    <label
-      data-testid="update-property-item-label"
-      class="input-item--label"
-    >
+  <div
+    data-testid="update-property-item-number-input-form"
```

Comment:
On second thought...

This is not _always_ a "number"-input-form...
The type of field is determined at line 41 below...

Reply:
Changed to: `update-property-item-input-form`